bin/newman.js

```diff
@@ -34,7 +34,7 @@ cli(process.argv.slice(2), 'newman', function (err, args) {
     dispatch(args, function (err) {
         if (err) {
             console.error(err);
-            process.exit(1);
+            process.exit(_.toNumber(_.get(args, 'run.suppressExitCode', true)));
```

Comment:
Currently, the behaviour of `lodash.toNumber` for `Boolean` values is identical to that of the `+` operator, so `true` is coerced to `1`, while `false` becomes `0`.